src/SS58Address.h

```diff
@@ -85,8 +91,45 @@ class SS58Address {
     }
 
     /// Returns public key bytes
-    Data keyBytes() const { 
-        return Data(bytes.begin() + 1, bytes.end());
+    Data keyBytes() const {
+        byte networkSize;
+        uint64_t networkTemp;
+        decodeNetwork(bytes, networkSize, networkTemp);
+        return Data(bytes.begin() + networkSize, bytes.end());
+    }
+
+    // Return true and the network size (1 or 2) and network if input is valid
+    static bool decodeNetwork(const Data& data, byte& networkSize, uint64_t& network) {
```

Comment:
Let's move `decodeNetwork` and `encodeNetwork` to `ScaleCodec.h`?

Reply:
These are a bit different than the encoding functions in ScaleCodec, and used only by the address.